### ISSUE TITLE
General configuration: PeriodicScript

### ISSUE THREAD

**Issue by Sadi58** [tubbadu/Scriptinator#5]
It seems a field for the "PeriodicScipt" is missing here, and also a box to set a time for this script to run every n seconds (like in the "Command Output" widget.

I'm guessing "Init script (to be run just once at the startup)" is not the "PeriodicScript, and "Timeout to run PeriodicScript (in seconds); set it to 0 to disable" (with a maximum value of 99 seconds) does not mean "PeriodicScript" will run every n seconds set here.

**Comment by tubbadu** [comment]
you're right, that's an error in the field description: it's not an icon, but a script! thanks for helping me making Scriptinator better!
